input/ez.raml

```diff
@@ -947,61 +947,61 @@ version: 2.5
+                        description: Created. If destination is '/api/ezp/v2/content/trash' and content only has one Location (NOTE - Like on normal subtree moves, be aware that the user might loose access to the item after it has been moved to trash.)
                     204:
-                        description: No Content. If destination is /api/ezp/v2/content/trash and content still has other locations (no trash item is created).
+                        description: No Content. If destination is '/api/ezp/v2/content/trash' and content still has other Locations (no trash item is created).
                     401:
-                        description: Error - the user is not authorized to move this location
+                        description: Error - the user is not authorized to move this Location.
                     404:
-                        description: Error - the Location with the given ID does not exist
+                        description: Error - the Location with the given ID does not exist.
             copy:
-                displayName: Copy Subtree
-                description: Copies the subtree to another parent. COPY or POST with header X-HTTP-Method-Override COPY
+                displayName: Copy subtree
+                description: Copies the subtree to a different parent. COPY or POST with header X-HTTP-Method-Override COPY.
                 headers:
                     Destination:
-                        description: A parent Location resource to which the Location is moved e.g. '/api/ezp/v2/content/locations/1/63'
+                        description: A parent Location resource to which the Location is moved e.g. '/api/ezp/v2/content/locations/1/63'.
                 responses:
                     201:
-                        description: Created. Copied the subtree to another parent.
+                        description: Created. Copied the subtree to a different parent.
                     401:
-                        description: Error - the user is not authorized to move this location
+                        description: Error - the user is not authorized to move this Location.
                     404:
-                        description: Error - the Location with the given ID does not exist
+                        description: Error - the Location with the given ID does not exist.
             delete:
-                displayName: Delete Subtree
-                description: Deletes the complete subtree for the given path. Every content object is deleted which does not have any other location. Otherwise the deleted Location is removed from the content object. The children are recursively deleted.
+                displayName: Delete subtree
+                description: Deletes the complete subtree for the given path. Every Content item, which does not have any other Location is deleted. Otherwise the deleted Location is removed from the Content item. The children are recursively deleted.
```

Comment:
```suggestion
                description: Deletes the complete subtree for the given path. Every Content item which does not have any other Location is deleted. Otherwise the deleted Location is removed from the Content item. The children are recursively deleted.
```